app/routes/AppContainer.jsx

```diff
@@ -0,0 +1,50 @@
+import React, { Component } from "react"
+import PropTypes from "prop-types"
+import { connect } from "react-redux"
+import { ipcRenderer } from "electron"
+
+import {assignmentFetchInfo} from "../modules/assignment/actions/assignment-fetch-info"
+import {setAssignmentURL} from "../modules/assignment/actions/assignment-set-url"
+import {settingsUpdateUserState} from "../modules/settings/actions/settings-update-user-state"
+
+class AppContainer extends Component {
+  constructor (props) {
+    super(props)
+
+    this.props.updateUserState() // Get user if already logged in from previous session
```

Comment:
Could this prop be renamed to indicate it's purpose and avoid the need for this comment? Something like `this.props.populateUserSession()`? This might also just be a case of taking time out to document new components...

Reply:
Added some documentation to the actual action so that it's more clear what this action does. I agree that `this.props.populateUserSession()` is a better name, but this code will most likely change depending on the security approach we go with. Can we do the name change in a future PR if we do end up using the session?